### PR TITLE
Ravinder | Test | Integrated angular material on daybook

### PR DESCRIPTION
**Pull request by ravinderwalkover**
https://app.clickup.com/t/7hhh87

### CODE REVIEW

File: apps/web-giddh/src/assets/css/style-1.scss [hunk 1]
```diff
@@ -2953,4 +2955,26 @@ update-ledger-entry-panel .ledger-main .mat-form-field {
         }
     }
 }
+.daybook-advance-search .mat-form-field-appearance-fill .mat-form-field-flex {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/css/style-1.scss:2958]: Consider nesting the classes within `daybook-advance-search`

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

**Review by prateek-giddh — APPROVED** [review]

File: apps/web-giddh/src/app/daybook/advance-search/daybook-advance-search.component.html [hunk 2]
```diff
@@ -327,7 +341,7 @@ <h3 class="modal-title bg" id="modal-title">{{ commonLocaleData?.app_advance_sea
                     </div>
                 </ng-container>
                 <div class="col-xs-12 col-sm-12 text-right">
-                    <button class="btn btn-success" type="button" (click)="go()">
+                    <button mat-raised-button color="primary" type="button" (click)="go()">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/daybook/advance-search/daybook-advance-search.component.html:344]: @ravindersinghdang please remove this color from html and copy the button class name properly.

File: apps/web-giddh/src/app/daybook/advance-search/daybook-advance-search.component.scss [hunk 3]
```diff
@@ -34,5 +33,11 @@ input.form-control.date-range-picker {
     border: 1px solid $chetwode-blue;
     border-radius: 4px;
     padding: 2px;
-    top: 0 !important;
+    top: 0px !important;
+}
+.giddh-datepicker.date-range-picker {
+    padding-left: 45px !important;
+}
+.modal-header > .close {
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/daybook/advance-search/daybook-advance-search.component.scss:41]: please add scss, not css @ravinderwalkover

File: apps/web-giddh/src/app/daybook/daybook.component.html [hunk 4]
```diff
@@ -167,138 +170,340 @@
...
-                            *ngFor="let entry of daybookData?.entries"
-                            (click)="expandEntry(entry)"
-                            class="daybook-table-tbody"
-                        >
-                            <tr
-                                (dblclick)="showUpdateLedgerModal(entry)"
-                                (keydown.enter)="showUpdateLedgerModal(entry)"
+                        <ng-container matColumnDef="expandedDetail">
+                            <mat-cell
+                                class="pd-0 w-100"
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/daybook/daybook.component.html:332]: @ravinderwalkover use class p-0, in place of pd-0 for padding 0 @ravinderwalkover

File: apps/web-giddh/src/app/daybook/export-daybook/export-daybook.component.html [hunk 5]
```diff
@@ -3,97 +3,46 @@
...
+                    <mat-radio-button id="ascending" [value]="'asc'"
+                        >{{ commonLocaleData?.app_ascending }}
+                    </mat-radio-button>
+                    <mat-radio-button id="descending" [value]="'desc'"
+                        >{{ commonLocaleData?.app_descending }}
+                    </mat-radio-button>
+                </mat-radio-group>
             </div>
-            <button class="btn-success btn" (click)="exportLedger()">{{ commonLocaleData?.app_download }}</button>
+            <button mat-raised-button color="primary" (click)="exportLedger()">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/daybook/export-daybook/export-daybook.component.html:43]: @ravinderwalkover remove the color primary and add the class which are used above @ravinderwalkover

File: apps/web-giddh/src/assets/css/style-1.scss [hunk 6]
```diff
@@ -2953,4 +2956,39 @@ update-ledger-entry-panel .ledger-main .mat-form-field {
...
+    }
+    .mat-form-field {
+        .mat-form-field-wrapper {
+            padding-bottom: 0px;
+        }
+    }
+}
+.export-daybook {
+    .mat-raised-button {
+        padding-left: 10px !important;
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/assets/css/style-1.scss:2987]: @ravinderwalkover we have classes pd-l1 and pd-r1 for this. please use that @ravinderwalkover

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 7]
```diff
@@ -9816,6 +9816,12 @@ body.update-scroll-hidden {
         width: auto;
         max-width: 300px;
     }
+    .d-sm-inline {
+        display: inline-block !important;
+    }
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/assets/css/style-2.scss:9821]: by-default class d-inline-block is used for display:inline-block thing, please use that. d-sm-inline is used for other css. @ravinderwalkover

File: apps/web-giddh/src/app/daybook/advance-search/daybook-advance-search.component.html (hunk 2 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/daybook/advance-search/daybook-advance-search.component.html:344]: Done

File: apps/web-giddh/src/app/daybook/advance-search/daybook-advance-search.component.scss (hunk 3 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/daybook/advance-search/daybook-advance-search.component.scss:41]: Done

File: apps/web-giddh/src/app/daybook/daybook.component.html (hunk 4 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/daybook/daybook.component.html:332]: Done

File: apps/web-giddh/src/app/daybook/export-daybook/export-daybook.component.html (hunk 5 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/daybook/export-daybook/export-daybook.component.html:43]: Done

File: apps/web-giddh/src/assets/css/style-1.scss (hunk 6 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/assets/css/style-1.scss:2987]: Done

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 7 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/assets/css/style-2.scss:9821]: Please check the logic also behind this not just name.

**Pull request merged**